<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRSettingRentgenAutoSetDialog</class>
 <widget class="QDialog" name="SPRSettingRentgenAutoSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>909</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter::handle:horizontal {
    width: 6px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(242, 242, 242), stop:1 rgb(149, 150, 149))
}

QSplitter::handle:vertical {
    height: 6px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(149, 150, 149), stop:1 rgb(242, 242, 242))
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="SPRSpectrumGraphicsWidget" name="graphic" native="true"/>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Текущие значения</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tDeuCP">
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>1</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Новая строка</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>2</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>4</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Ручей</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>ДЭУ</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>ЦП</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tPeakRS">
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>Новая строка</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>1</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>2</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>4</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Ручей</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Пик</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>RS</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Контроль выполнения, %</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cbStartWithCurrentValues">
               <property name="text">
                <string>Стартовать со значений из настройки</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbSuspendMode">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Продолжить выполнение</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/icons/icons/b_Resume.png</normaloff>:/icons/icons/b_Resume.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>Привести пик спектра к каналу:</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QSpinBox" name="spPeakPosition"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QGroupBox" name="gbControlCP">
                  <property name="title">
                   <string>ЦП</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QwtThermo" name="tmContrilCP">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="scalePosition">
                      <enum>QwtThermo::NoScale</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGroupBox" name="gbControlDEU">
                     <property name="title">
                      <string>ДЭУ</string>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QwtThermo" name="tmContrilDEU">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="scalePosition">
                         <enum>QwtThermo::NoScale</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QGroupBox" name="gbControlAll">
                        <property name="title">
                         <string>Процесс</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_6">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <property name="leftMargin">
                          <number>0</number>
                         </property>
                         <property name="topMargin">
                          <number>0</number>
                         </property>
                         <property name="rightMargin">
                          <number>0</number>
                         </property>
                         <property name="bottomMargin">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QwtThermo" name="tmContrilAll">
                           <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                           </property>
                           <property name="scalePosition">
                            <enum>QwtThermo::NoScale</enum>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QWidget" name="widget_3" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="bStart">
                     <property name="text">
                      <string>Старт</string>
                     </property>
                     <property name="icon">
                      <iconset resource="icons.qrc">
                       <normaloff>:/icons/icons/b_Flag.png</normaloff>:/icons/icons/b_Flag.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bRentgenOff">
                     <property name="text">
                      <string>Выключить сепаратор</string>
                     </property>
                     <property name="icon">
                      <iconset resource="icons.qrc">
                       <normaloff>:/icons/icons/Separate_Close_tab_icon.png</normaloff>:/icons/icons/Separate_Close_tab_icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_5" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QPushButton" name="bComplite">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>Применить</string>
                     </property>
                     <property name="icon">
                      <iconset resource="icons.qrc">
                       <normaloff>:/icons/icons/b_Ok_1.png</normaloff>:/icons/icons/b_Ok_1.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bResume">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>Продолжить</string>
                     </property>
                     <property name="icon">
                      <iconset resource="icons.qrc">
                       <normaloff>:/icons/icons/b_Resume.png</normaloff>:/icons/icons/b_Resume.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bExit">
                     <property name="text">
                      <string>Выход</string>
                     </property>
                     <property name="icon">
                      <iconset resource="icons.qrc">
                       <normaloff>:/icons/icons/b_Exit.png</normaloff>:/icons/icons/b_Exit.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
  <customwidget>
   <class>SPRSpectrumGraphicsWidget</class>
   <extends>QWidget</extends>
   <header>sprspectrumgraphicswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbStartWithCurrentValues</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbSuspendMode</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSuspendMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbStartWithCurrentValues</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbStartWithCurrentValues</sender>
   <signal>toggled(bool)</signal>
   <receiver>bResume</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
